### ISSUE TITLE
"vector too long" when building and running tests with cmake on Windows using GitHub actions

### ISSUE THREAD

**Issue by cassiebeckley** [KhronosGroup/SPIRV-Reflect#167]
The errors look like

```
[ RUN      ] SpirvReflectMultiEntryPointTest.GetEntryPoint
unknown file: error: C++ exception with description "vector too long" thrown in SetUp().
```

I haven't been able to reproduce this on my own Windows machine.

**Comment by XDzzzzzZyq** [comment]
Hi, have you solved this problem? I encountered the same issue like this

**Comment by cassiebeckley** [comment]
No, it hasn't been solved yet. I'll try and look more into it. Would you be able to share which version of Windows and Visual Studio you are using?

**Comment by XDzzzzzZyq** [comment]
> which version of Windows and Visual Studio you are using?

Windows11 
Visual studio 2019

**Comment by Keenuts** [comment]
Was able to reproduce this issue on a windows machine, but also linux.
The reason is the filepath used to fetch the test content expects us to run the test from the test directory.
If we run it from the wrong directory, the file fails to open, and `tellg` returns `-1`, which is then passed to `vector::resize` without checking, causing the `vector too long` error.
